WooCommerce/Classes/ViewRelated/Orders/OrdersSplitViewWrapperController.swift

```diff
@@ -61,6 +61,15 @@ private extension OrdersSplitViewWrapperController {
         ordersSplitViewController.viewControllers = [ordersNavigationController, emptyStateViewController]
     }
 
+    func handleCollapsingSplitView(splitViewController: UISplitViewController) -> UISplitViewController.Column {
```

Comment:
I wonder if it's worth adding tests for this behavior 🤔

Reply:
Yes, it'd be nice to add tests but I'm not sure yet how to simulate split view's collapsed mode in tests 😅 I'm not so good at writing unit tests for UI code, but I'll add an item in the checklist for that to take a closer look later.